src/Http/Middleware/HttpsEnforcerMiddleware.php

```diff
@@ -46,6 +53,7 @@ class HttpsEnforcerMiddleware implements MiddlewareInterface
         'statusCode' => 301,
         'headers' => [],
         'disableOnDebug' => true,
+        'hsts' => false,
```

Comment:
Since a valid value is an array having default as `null` makes more sense. That will also make it more obvious that `true` isn't a valid value for this config.

Reply:
If we use array|null, do we still need to check the type? It will throw a php error.

Reply:
Explicit check allow providing a more user friendly error message but personally I am fine with a PHP error too.

Reply:
Switched to null.